app/routes/candidates.js

```diff
@@ -6,7 +6,7 @@ var db = new sqlite3.Database('db');
 /* GET users listing. */
 router.get('/', function(req, res, next) {
   var candidates = [];
-  var selectClause = 'SELECT firstName, lastName, party, chamber, state, district, incumbent, website, email, facebook, twitter, youtube FROM candidates';
+  var selectClause = 'SELECT img_src, firstName, lastName, party, chamber, state, district, incumbent, website, email, facebook, twitter, youtube FROM candidates';
```

Comment:
I think the keys are translated to the members of candidate, so the order shouldn't be a problem. It works on my machine.